pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -381,7 +381,7 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 				klog.V(2).Infof("Finish parsing log file %q, hit bytes limit %d(bytes)", path, opts.bytes)
 				return nil
 			}
-			klog.Errorf("Failed with err %v when writing log for log file %q: %+v", err, path, msg)
+			klog.Errorf("Failed with err %v when writing log for log file %q: %q", err, path, msg)
```

Comment:
govet check failed with error  " `Errorf format %q has arg msg of wrong type`" @utkarshmani1997

Reply:
Since the purpose of msg is to print the logs, shall i only print the string(msg.log) ?